ENV prod/dev mode and start_wmm_tracking ownerid improvement

durzo opened the pull request:
-  introduces a new ENV variable in the .env file which defaults to 'prod' if not specified. If this is set to 'dev' then additional debugging is activated and low stock DMs are overridden and sent only to DEVOWNERID. Nothing needs to be changed in the .env file for production, this is purely for my local environment.

- Makes the 'owner' parameter of _start_wmm_tracking_ optional, if not provided it will now take the ID of the user who typed the command.

durzo committed:
ENV prod/dev mode and start_wmm_tracking ownerid improvement

gcruicks commented on bot.py:
```diff
@@ -24,38 +24,37 @@
+    logger.setLevel(logging.DEBUG)
+    handler = logging.FileHandler(filename='discord.log', encoding='utf-8', mode='w')
+    handler.setFormatter(logging.Formatter('%(asctime)s:%(levelname)s:%(name)s: %(message)s'))
+    logger.addHandler(handler)
```
We do have this, that might not be published to pypi today, but is usable:  https://github.com/PilotsTradeNetwork/PTNLogger

I would love at some point to see us actually use this instead of the print statements everywhere

gcruicks replied:
Actually, do you ever use this logger in the code?

durzo replied:
this code writes all the discordpy debug output to the discord.log file and is very handy to see the events and contexts.

durzo replied:
it is not a logger for my own code, and i dont intend to use it that way

gcruicks replied:
Ah. Now I follow. Might be worth adding a comment that you are adding a handler to discords internal logger.

durzo replied:
I got this snippet from the discordpy docs here https://discordpy.readthedocs.io/en/stable/logging.html

Happy to add a comment in a later commit

durzo committed:
Fix bug if no carriers have stock in a station

durzo committed:
Add comments to discord debug handler

durzo commented:
Fixed the bug that occured on Saturday where if all carriers in a station have no stock, the dict key was missing and an exception was thrown.

Also added comments to debug handler as requested

durzo committed:
Update inara functions to use new URL as per request

durzo commented:
Changed the Inara URLs as requested so we don't get blocked again.
This will buy us time until January when we need to launch cAPI.
